src/Magnum/MeshTools/Test/BoundingVolumeTest.cpp

```diff
@@ -0,0 +1,217 @@
+        const Containers::Pair<Vector3, Float> sphere =
+            MeshTools::boundingSphereBouncingBubble(sphereMesh.attribute<Vector3>(Trade::MeshAttribute::Position));
+        /* No error */
+        CORRADE_COMPARE(sphere.first(), (Vector3{0.0f, 0.0f, 0.0f}));
+        CORRADE_COMPARE(sphere.second(), 1.0f);
+    } {
+        Trade::MeshData sphereMesh = Primitives::icosphereSolid(1);
+        constexpr Vector3 translation{1.0f, 2.0f, 3.0f};
+        constexpr Vector3 scale{0.5f, 1.2f, 2.8f};
+        MeshTools::transformPointsInPlace(Matrix4::translation(translation)*Matrix4::scaling(scale),
+            sphereMesh.mutableAttribute<Vector3>(Trade::MeshAttribute::Position));
+        const Containers::Pair<Vector3, Float> sphere =
+            MeshTools::boundingSphereBouncingBubble(sphereMesh.attribute<Vector3>(Trade::MeshAttribute::Position));
+        /* Noticeable error */
+        constexpr Float Delta = 0.04f;
+        CORRADE_COMPARE_WITH(sphere.first(), translation, TestSuite::Compare::Around<Vector3>{Vector3{Delta}});
+        /* Radius should never be smaller than the ground truth */
+        CORRADE_COMPARE_WITH(sphere.second(), scale.max() + Delta, TestSuite::Compare::Around<Float>{Delta});
+
+    /* Cube -- translated and scaled */
+    } {
+        Trade::MeshData cubeMesh = MeshTools::owned(Primitives::cubeSolid());
+        constexpr Vector3 translation{1.0f, 2.0f, 3.0f};
+        constexpr Float scale = 13.2f;
+        MeshTools::transformPointsInPlace(Matrix4::translation(translation)*Matrix4::scaling(Vector3{scale}),
+            cubeMesh.mutableAttribute<Vector3>(Trade::MeshAttribute::Position));
+        const Containers::Pair<Vector3, Float> sphere =
+            MeshTools::boundingSphereBouncingBubble(cubeMesh.attribute<Vector3>(Trade::MeshAttribute::Position));
+        /* Noticeable error */
+        constexpr Float Delta = 0.04f;
+        CORRADE_COMPARE_WITH(sphere.first(), translation, TestSuite::Compare::Around<Vector3>{Vector3{Delta}});
+        CORRADE_COMPARE_WITH(sphere.second(), Math::sqrt(3.0f)*scale + Delta,
+            TestSuite::Compare::Around<Float>{Delta});
+    }
+
+    /* Radius is rotationally invariant */
+    using namespace Math::Literals;
+
+    for(Deg degrees = 0.0_degf; degrees < 360.0_degf; degrees += 60.0_degf) {
+        CORRADE_ITERATION(degrees);
+        Trade::MeshData cubeMesh = MeshTools::owned(Primitives::cubeSolid());
+        constexpr Vector3 translation{1.0f, 2.0f, 3.0f};
+        MeshTools::transformPointsInPlace(Matrix4::rotationY(degrees)*Matrix4::translation(translation),
+            cubeMesh.mutableAttribute<Vector3>(Trade::MeshAttribute::Position));
+        const Containers::Pair<Vector3, Float> sphere =
+            MeshTools::boundingSphereBouncingBubble(cubeMesh.attribute<Vector3>(Trade::MeshAttribute::Position));
+        CORRADE_COMPARE(sphere.second(), Math::sqrt(3.0f));
+    }
+}
+
+void BoundingVolumeTest::benchmarkSphereBouncingBubble() {
```

Comment:
Just to have a reference point to compare this to, maybe add a second benchmark with the ~~minmax~~ AABB?